overview/requirements.md

```diff
@@ -77,15 +77,15 @@ needs to be.
 
 Every role will have different required software, but it is recommended
 for every role to have installed [Aegisub][]. It
-is **highly** recommended to use [line0's builds][]. They include a
-preinstalled automation script package manager and several *critical*
-fixes to Aegisub that have not been merged into the official
-application.
+is **highly** recommended to use [line0's builds][].
+They come pre-equipped with Dependency Control
```

Comment:
Yes, it's in GLOSSARY.md . If this version of Gitbook is documented correctly, it should become a highlighted phrase that when you mouse over explains what it is with a link to the section in the glossary. This was mainly a test of the feature, but I could see it being used in the future to avoid having footnotes and redundancy when explaining something off-topic.